docs/guides/private-python-packages.rst

```diff
@@ -0,0 +1,108 @@
+Using Private Python Packages with Read the Docs for Business
+=============================================================
+
+Read the Docs uses :doc:`PIP <pip:index>` to install your Python packages,
+see :doc:`/guides/specifying-dependencies`.
+
+Installing packages from a Git repository
+-----------------------------------------
+
+PIP supports installing packages from a :ref:`Git repository <pip:vcs support>` using the form:
+
+.. code::
+   
+   git+https://gitprovider.com/user/project.git@{version}#egg={package-name}
+
+Where ``version`` can be a tag, a branch, or a commit.
+
+If your repository is private you need to use the form:
+
+.. code::
+   
+   git+https://{token}@gitprovider.com/user/project.git@{version}#egg={package-name}
+
+Where ``token`` is a personal access token with read only permissions from your provider.
+Below you can find the steps to get a personal token from our supported providers.
+
+.. We should add the git+ssh form when we support running the ssh-agent in the build step.
+
+.. note::
+   
+   The URI should be in your :ref:`requirements file <config-file/v2:Requirements file>`.
+   PIP will automatically expand environment variables in your URI,
+   you can :doc:`add environment variables </guides/environment-variables>` in Read the Docs,
+   so you don't to hard code the token in the URI.
```

Comment:
I didn't check this, but could this be used to add a private dependency on readthedocs.org?

Reply:
Yes, it can be used to install private packages in .org, Not sure if users will use this in .org,  if you have a private dependency your repo should be private too?